modules/nvidia-gpu-vsphere.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-vsphere_{context}"]
+= GPUs and vSphere
+
+You can deploy {product-title} on an NVIDIA-certified VMware vSphere server that can host different GPU types.
+
+An NVIDIA GPU driver must be installed in the hypervisor in case vGPU instances are used by the VMs. For VMWare vSphere, this host driver is provided in the form of a VIB file.
+
+The maximum number of vGPUS that can be allocated to worker node VMs depends on the version of vSphere:
+
+* vSphere 7.0: maximum 4 vGPU max per VM
+* vSphere 8.0: maximum 8 vGPU max per VM
```

Comment:
Is this redundant? 

* vSphere 7.0: maximum 4 vGPU ~~max~~ per VM
* vSphere 8.0: maximum 8 vGPU ~~max~~ per VM